src/tests/ftest/pool/info_tests.py

```diff
@@ -73,7 +73,7 @@ def test_pool_info_query(self):
         self.assertTrue(status, "Invlaid pool information detected prior")
         checks = {
             "s_total": (self.pool.scm_size.value, 0),
-            "s_free": (self.pool.scm_size.value - (256 * pool_targets), 0),
+            #"s_free": (self.pool.scm_size.value - (256 * pool_targets), 0),
```

Comment:
This test needs be improved, please make sure there is an open ticket to track this.

Reply:
I created one for it